src/helpers/helpers.config.ts

```diff
@@ -176,11 +211,15 @@ export function _descriptors(proxy, defaults = {scriptable: true, indexable: tru
   };
 }
 
-const readKey = (prefix, name) => prefix ? prefix + _capitalize(name) : name;
-const needsSubResolver = (prop, value) => isObject(value) && prop !== 'adapters' &&
+const readKey = (prefix: string, name: string) => prefix ? prefix + _capitalize(name) : name;
+const needsSubResolver = (prop: string, value: any) => isObject(value) && prop !== 'adapters' &&
```

Comment:
A thought here. Is `unknown` better than `any`? I recall that `any` disables some TS features whereas `unknown` does not